content/en/blogs/TiDB-on-KubeSphere-using-qke.md

```diff
@@ -56,7 +58,7 @@ kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/v1.1.6/
 
 ![add-pingcap-repo](https://ap3.qingstor.com/kubesphere-website/docs/20201026193015.png)
 
-## Deploying tidb-operator
+## Deploying TiDB-operator
```

Comment:
I did not capitalize t in tidb as the app is named "tidb-operator" in the repo.

![Uploading image.png…]()